R/bm-write-csv.R

```diff
@@ -0,0 +1,74 @@
+#' Benchmark CSV writing
+#'
+#' @section Parameters:
+#' * `source` A CSV file path to write to
+#' * `writer` One of `c("arrow", "data.table", "vroom", "readr",)`
+#' * `input` One of `c("arrow_table", "data_frame")`
+#'
+#' @export
+write_csv <- Benchmark(
+  "write_csv",
+  setup = function(source = names(known_sources),
+                   writer = c("arrow", "data.table", "vroom", "readr", "utils"),
```

Comment:
```suggestion
                   writer = "arrow",
```

This is super subtle (and not super well documented) but what's listed here will be what's run by default if this isn't otherwise specified. So using only `"arrow"` will mark that as the default, then we add the others to `match.arg` below

Reply:
I actually _did_ read that but it didn't sink in until right now. Thanks.